packages/extensions/modelerfour/test/modeler/modelerfour.requests.test.ts

```diff
@@ -234,4 +235,141 @@ describe("Modelerfour.Request", () => {
+                },
+              },
+            },
+            "202": {
+              description: "Accepted",
+            },
+          },
+        },
+      });
+      addSchema(oldSpec, "Product", {
+        type: "object",
+        properties: {
+          id: {
+            type: "string",
+          },
+        },
+      });
+
+      const customizer = (spec: any): any => {
+        const operation = spec.paths["/products/{id}"]["put"];
+        delete operation["responses"]["200"];
+        operation["x-ms-long-running-operation-options"]["final-state-schema"] = "#/components/schemas/Product";
+        return spec;
+      };
+
+      const newSpec = customizer(cloneDeep(oldSpec));
+
+      const oldModel = await runModeler(oldSpec);
+      const newModel = await runModeler(newSpec);
+      const oldResponses = oldModel.operationGroups[0].operations[0].responses;
+      const newResponses = newModel.operationGroups[0].operations[0].responses;
+      expect(oldResponses).toBeDefined();
+      expect(newResponses).toBeDefined();
+      expect(newResponses).toEqual(oldResponses);
+    });
+
+    it("does not modify 200 response from final-state-schema if one is present", async () => {
+      const oldSpec: oai3.Model = createTestSpec();
+      addOperation(oldSpec, "/products/{id}", {
+        put: {
+          operationId: "product_create",
+          description: "Create a Product",
+          "x-ms-long-running-operation": true,
+          "x-ms-long-running-operation-options": {
+            "final-state-via": "location",
+          },
+          responses: {
+            "200": {
+              description: "Success",
+              content: {
+                "application/json": {
+                  schema: { $ref: "#/components/schemas/Product" },
+                },
+              },
+            },
+            "202": {
+              description: "Accepted",
+            },
+          },
+        },
+      });
+      addSchema(oldSpec, "Product", {
+        type: "object",
+        properties: {
+          id: {
+            type: "string",
+          },
+        },
+      });
+
+      const customizer = (spec: any): any => {
+        const operation = spec.paths["/products/{id}"]["put"];
+        operation["x-ms-long-running-operation-options"]["final-state-schema"] = "#/components/schemas/Widget";
+        return spec;
+      };
+      const newSpec = customizer(cloneDeep(oldSpec));
+
+      const oldModel = await runModeler(oldSpec);
+      const newModel = await runModeler(newSpec);
+      const oldResponses = oldModel.operationGroups[0].operations[0].responses;
+      const newResponses = newModel.operationGroups[0].operations[0].responses;
+      expect(oldResponses).toBeDefined();
+      expect(newResponses).toBeDefined();
+      expect(newResponses).toEqual(oldResponses);
+    });
+
+    // Modelerfour allows an LRO to have a 202 response without a 200 response.
```

Comment:
>202 [](http://example.com/codeflow?start=43&length=3)

applies to 201 as well right?

Reply:
Right ... neither 200 nor 201 is needed. I'll fix.